cli.js

```diff
@@ -358,12 +358,17 @@ process.on('uncaughtException', function(err) {
       var sfxBundle = true;
 
     case 'bundle':
-      options = readOptions(args, ['--inject', '--yes', '--skip-source-maps', '--minify',  '--no-mangle', '--hires-source-maps', '--no-runtime']);
+      options = readOptions(args, ['--inject', '--yes', '--skip-source-maps', '--minify',  '--no-mangle', '--hires-source-maps', '--no-runtime', '--inline-source-maps']);
       if (options.yes)
         ui.useDefaults();
       options.sourceMaps = !options['skip-source-maps'];
       options.lowResSourceMaps = !options['hires-source-maps'];
       options.mangle = !options['no-mangle'];
+      
+      if (options['inline-source-maps']){
+        options.sourceMapContents = 'inline';
```

Comment:
This sounds like an error in the CSS plugin due to https://github.com/systemjs/plugin-css/blob/master/css-builder.js#L32 passing in the full value of `inline`.

Setting `sourceMapContents: true` seems like a viable alternative, but it should be `true` and not `inline` I think.

In terms of the API, we should probably try and converge on one or the other method though. I quite like the separate property actually to avoid issues like that.